packages/app-store/routing-forms/trpc/response.handler.ts

```diff
@@ -93,7 +94,31 @@ export const responseHandler = async ({ ctx, input }: ResponseHandlerOptions) =>
       data: input,
     });
 
-    await onFormSubmission(serializableFormWithFields, dbFormResponse.response as Response);
+    const settings = RoutingFormSettings.parse(form.settings);
+    let userWithEmails: string[] = [];
+    if (form.teamId && settings?.sendUpdatesTo?.length) {
+      const userEmails = await prisma.membership.findMany({
+        where: {
+          teamId: form.teamId,
+          userId: {
+            in: settings.sendUpdatesTo,
+          },
+        },
+        include: {
```

Comment:
We should just only return the user email

Reply:
Done